EMML/6001-7000/EMML6451.xml

```diff
@@ -30,89 +30,49 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                                 <locus from="2ra" to="113rb"/>
                                 <title type="complete" ref="LIT1470GadlaL"/>
-                                <msItem xml:id="ms_i1.1">
-                                    <locus from="2ra" to="7va"/>
-                                    <title type="complete" ref="LIT5080TUoneLal"/>
-                                    <incipit xml:lang="gez">
-                                    </incipit>
-                                </msItem>
-                      
-                            <msItem xml:id="ms_i1.2">
-                                <locus from="7va" to="11rb"/>
-                                <title type="complete" ref="LIT5081SoterLal"/>
-                            </msItem>
-                            <msItem xml:id="ms_i1.3">
-                                <locus from="11rb" to="13vb"/>
-                                
-                                <title type="complete" ref="LIT5082TUthreeLal"/>
-                            </msItem>
-                            <msItem xml:id="ms_i1.4">
-                                <locus from="13vb" to="15va"/>
-                                <title type="incomplete" ref="LIT5083TUfourLal"/>
-                            </msItem>
-                            <msItem xml:id="ms_i1.5">
-                                <locus from="15va" to="25ra"/>
-                                <title type="incomplete" ref="LIT5086TUsixLal"/>
-                            </msItem>
-                            
-                            <msItem xml:id="ms_i1.6">
-                                <locus from="25rb" to="41va"/>
-                                <title type="incomplete" ref="LIT4922HagVit"/>
-                                <incipit/>
-                            </msItem>
-                            <msItem xml:id="ms_i1.7">
-                                <locus from="41va" to="44va"/>
-                                <title type="incomplete" ref="LIT5083TUfourLal"/>
-                            </msItem>
-                            <msItem xml:id="ms_i1.8">
-                                <locus from="44va" to="47vb"/>
-                                <title type="complete" ref="LIT5084TUfiveLal"/>
-                            </msItem>
-                            <msItem xml:id="ms_i1.9">
-                                <locus from="47vb" to="49va"/>
-                                <title type="incomplete" ref="LIT5086TUsixLal"/>
-                            </msItem>
-                            
-                            <msItem xml:id="ms_i1.10">
-                                <locus from="49va" to="100rb"/>
-                                <title type="incomplete" ref="LIT4922HagVit"/>
-                                <incipit/>
-                            </msItem>
-                            <msItem xml:id="ms_i1.11">
+                            <msItem xml:id="ms_i1.1">
```

Comment:
I assume all miracles should be subitems to the taamrat item ms_i1.2, and ms_i1.1 should be reinserted as the gadl (?) to cover folia from 1 to 100, please check carefully all numberings and nestings here and elsewhere, maybe some of the deleted information should be reinserted and missing work items recreated. thank you